src/operations/blas3/gemm_local.hpp

```diff
@@ -228,35 +217,42 @@ class Gemm<input_t, output_t, DoubleBuffer, NbcA, NbcB, ClSize, TileType,
   template <typename local_memory_t>
   SYCL_BLAS_INLINE void eval(local_memory_t scratch_acc,
                              cl::sycl::nd_item<1> id) noexcept {
```

Comment:
here as well change it to either cl::sycl::nd_item<1>& or const cl::sycl::nd_item<1>&

Reply:
Fixed!